app/javascript/pages/account/components/DOBFieldset.tsx

```diff
@@ -166,6 +161,7 @@ const DateField = ({
         },
       }}
       inputProps={{ maxLength: fieldInfo[fieldKey].maxLength }}
+      type="number"
```

Comment:
issue: `e`, `-`, `+`, and `.` can be entered. Use [pattern attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern). Also use `aria-label` attribute for a11y.

Reply:
I remember discussing the character issue in a meeting and, as this would require an update to the bloom component, I thought we decided not to worry about this right now. We should create a follow-up ticket and request bloom uic to be updated.

+1 to the aria-label

Reply:
@chadbrokaw you can pass in a pattern though inputProps

Reply:
@jimlin-sfgov did you try implementing this? You can plug a pattern into the inputProps, but it doesn't seem to prevent the key from being entered. I think we should address this in another ticket

+1 to the aria label though

Reply:
Good catch! Please correct me if I'm misunderstanding, but the pattern attribute does not prevent characters from being entered, so adding that would be redundant with the validation check being run through React Hook Form?

Reply:
> did you try implementing this?

No I didn't, thanks for calling me out!